src/cmd/src/datanode.rs

```diff
@@ -99,15 +98,16 @@ struct StartCommand {
     http_addr: Option<String>,
     #[clap(long)]
     http_timeout: Option<u64>,
+    #[clap(long, default_value = "DATANODE")]
```

Comment:
Consider add prefix `GREPTIMEDB_`

Reply:
@sunng87 you mean all the env prefixes should be `GREPTIMEDB_`? If so, looks more reasonable and easy to use.

Reply:
Oh, I got it. You mean the format like `GREPTIMEDB_DATANODE_*`.